resources/i18n/zh_CN/cura.po

```diff
@@ -0,0 +1,3706 @@
+#, python-brace-format
+msgctxt "@info:status"
+msgid ""
+"Failed to export profile to <filename>{0}</filename>: Writer plugin reported "
+"failure."
+msgstr "无法将配置文件导出为<filename> {0} </filename>：插件返回失败。"
+
+#: /home/ruben/Projects/Cura/cura/Settings/CuraContainerRegistry.py:151
+#, python-brace-format
+msgctxt "@info:status"
+msgid "Exported profile to <filename>{0}</filename>"
+msgstr "导出的配置文件为：<filename> {0} </filename>"
+
+#: /home/ruben/Projects/Cura/cura/Settings/CuraContainerRegistry.py:177
+#: /home/ruben/Projects/Cura/cura/Settings/CuraContainerRegistry.py:199
+#: /home/ruben/Projects/Cura/cura/Settings/CuraContainerRegistry.py:208
+#: /home/ruben/Projects/Cura/cura/Settings/CuraContainerRegistry.py:242
+#, python-brace-format
+msgctxt "@info:status"
+msgid ""
+"Failed to import profile from <filename>{0}</filename>: <message>{1}</"
+"message>"
+msgstr ""
+"无法从 <filename> {0} </filename> 导入配置文件：<message> {1} </message>"
+
+#: /home/ruben/Projects/Cura/cura/Settings/CuraContainerRegistry.py:210
+#: /home/ruben/Projects/Cura/cura/Settings/CuraContainerRegistry.py:246
+#, python-brace-format
+msgctxt "@info:status"
+msgid "Successfully imported profile {0}"
+msgstr "已成功导入配置：{0}"
+
+#: /home/ruben/Projects/Cura/cura/Settings/CuraContainerRegistry.py:249
+#, python-brace-format
+msgctxt "@info:status"
+msgid "Profile {0} has an unknown file type or is corrupted."
+msgstr "配置 {0} 文件类型未知或已损坏。"
+
+#: /home/ruben/Projects/Cura/cura/Settings/CuraContainerRegistry.py:267
+msgctxt "@label"
+msgid "Custom profile"
+msgstr "自定义配置"
+
+#: /home/ruben/Projects/Cura/cura/Settings/CuraContainerRegistry.py:278
+msgctxt "@info:status"
+msgid "Profile is missing a quality type."
+msgstr "配置文件缺少「打印质量」类型定义。"
+
+#: /home/ruben/Projects/Cura/cura/Settings/CuraContainerRegistry.py:300
+#, python-brace-format
+msgctxt "@info:status"
+msgid "Could not find a quality type {0} for the current configuration."
+msgstr "﻿无法为当前配置找到质量类型 {0}。"
+
+#: /home/ruben/Projects/Cura/cura/BuildVolume.py:95
+msgctxt "@info:status"
+msgid ""
+"The build volume height has been reduced due to the value of the \"Print "
+"Sequence\" setting to prevent the gantry from colliding with printed models."
+msgstr ""
+"由于“打印序列”设置的值，建立体积高度已被减少，以防止龙门架与打印机型相冲突。"
+
+#: /home/ruben/Projects/Cura/cura/MultiplyObjectsJob.py:34
+msgctxt "@info:status"
+msgid "Multiplying and placing objects"
+msgstr "复制并放置模型"
+
+#: /home/ruben/Projects/Cura/cura/CrashHandler.py:54
+msgctxt "@title:window"
+msgid "Crash Report"
+msgstr "错误报告"
+
+#: /home/ruben/Projects/Cura/cura/CrashHandler.py:79
+msgctxt "@label"
+msgid ""
+"<p>A fatal exception has occurred that we could not recover from!</p>\n"
+"        <p>Please use the information below to post a bug report at <a href="
+"\"http://github.com/Ultimaker/Cura/issues\">http://github.com/Ultimaker/Cura/"
+"issues</a></p>\n"
+"    "
+msgstr ""
+"<p>发生了致命错误，我们无法恢复！</p>\n"
+"        <p>请在以下网址中提交错误报告：<a href=“http://github.com/Ultimaker/"
```

Comment:
There double quotes here for `<a href>` are still not ASCII quotes. Please change to ASCII quotes `"` like the ones in `msgid` otherwise the link won't work...